app/src/main/java/com/github/featuredetectandroid/ui/Menu.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.featuredetectandroid.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.github.featuredetectandroid.ui.theme.Typography
+import com.github.featuredetectandroid.utils.algorithmsNamesAsList
+
+@Composable
+fun Menu(currentAlgorithm: String, changeSelectedAlgorithmIntPreferences: (String) -> Unit) {
+    val radioOptions = algorithmsNamesAsList()
+    val (selectedAlgorithm, onAlgorithmSelection) = remember { mutableStateOf(currentAlgorithm) }
+
+    Row(horizontalArrangement = Arrangement.Center, verticalAlignment = Alignment.Bottom) {
+        Text(
+            text = "Keypoint detection algorithm:",
+            modifier = Modifier.padding(all = 20.dp),
+            textAlign = TextAlign.Start,
+            style = Typography.body1
+        )
+    }
+    radioOptions.forEach { text ->
+        Row(
+            Modifier
+                .fillMaxWidth()
+                .selectable(
+                    selected = (text == selectedAlgorithm),
+                    onClick = {
+                        onClickChangeSelectedAlgorithm(
+                            changeSelectedAlgorithmIntPreferences,
+                            onAlgorithmSelection,
+                            text
+                        )
+                    }
+                ),
+            horizontalArrangement = Arrangement.Start,
+            verticalAlignment = CenterVertically
+        ) {
+            RadioButton(
+                selected = (text == selectedAlgorithm),
+                modifier = Modifier.padding(horizontal = 8.dp),
+                onClick = {
+                    onClickChangeSelectedAlgorithm(
+                        changeSelectedAlgorithmIntPreferences,
+                        onAlgorithmSelection,
+                        text
+                    )
+                }
```

Comment:
Do we really need this `onClick` if the whole `Row` is clickable? Why not ust set it to `null`? You won't need this `onClickChangeSelectedAlgorithm` this way.

Reply:
Yes, because otherwise clicking on the button (namely, the round button) will not have any effect, but clicking on the row will. That is, as I understood from small experiments, clicking on a row works on the entire area, except for the button, and clicking on the button works only for the button itself.

Reply:
When I set `RadioButton`'s `onClick` to `null` everything works as expected: the whole row is clickable including the button

Reply:
Padding needs to be adjusted though, because the button itself becomes unclickable and Compose removes the padding it adds to clickable elements

Reply:
fixed